recipes/metacerberus-lite/meta.yaml

```
@@ -1,20 +1,20 @@
 {% set name = "MetaCerberus-lite" %}
-{% set version = "1.3.1" %}
+{% set version = "1.3.2" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: "https://files.pythonhosted.org/packages/source/m/MetaCerberus/MetaCerberus-{{ version }}.tar.gz"
-  sha256: 725be35a0e9b2a3867791e1cb8284ec95a09d3479516e9b265e3751659ce7935
+  url: "https://github.com/raw-lab/MetaCerberus/archive/refs/tags/v{{version}}.tar.gz"
+  sha256: 678929f58b38fafc635dfb35e248b5c76b4cd682b9e2d3499fe953ee3b974d68
 
 build:
   number: 0
   noarch: python
   script: "{{ PYTHON }} -m pip install . -vv"
   run_exports:
-    - {{ pin_compatible('MetaCerberus', max_pin="x") }}
+    - {{ pin_compatible('MetaCerberus-lite', max_pin="x") }}
```

```suggestion
    - {{ pin_subpackage('metacerberus-lite', max_pin="x") }}
```
Not sure if the previous version was trying to do something special with the non-lite MetaCerberus or if it was just a mistake. Bioconda docs recommend against using `pin_compatible`. https://bioconda.github.io/contributor/linting.html#missing-run-exports

---

Thank you for your feedback! I'm still a little confused on these run_exports and pinning, the previous version was not trying to do anything with the non-lite version of MetaCerberus, I saw that I had not updated that line and intended to fix it. Your suggestion makes sense to me.